src/styles/blocks/cards-container.scss

```diff
@@ -0,0 +1,18 @@
+$paddingX: 40px;
+
+@function getWidth($card-width) {
+  @return 4 * $card-width + 2 *$paddingX + 3 * 48;
+}
```

Comment:
It's better to create a new file _functions.scss and put your functions there, so it can be easier for you to find them